### DISCUSSION TITLE
Filtering through books on main cookbook page

### DISCUSSION THREAD

**Discussion by IxsharpxI** [TandoorRecipes/recipes#530] [Ideas]
So im not sure if there is a feature request area but i thought that this may be useful for some people with multiple books. I chose books over spaces because i wanted to share things between users. but anyway. Would it be helpful for anyone else to just have a drop down at the top near the search bar on the main cookbook page where you could filter by books? then if i chose book 1 is would list all the recipes/recents from book one.  And also having an "all" or "not in a book" options as well.

currently the only way to see only book 1 is to go to books and "toggle" the recipes. this is pretty limiting. and couldnt search using the method well. 

open to any input. thanks for all the hard work.

**Comment by vabene1111** [comment]
thanks for the feedback, my brother requested the exact same feature and i will add it when i am redoing the search page. Will probably be a little bit but its definitely planned

**Reply by IxsharpxI** [reply]
thanks for all your hardwork and communication!
